google-cloud-tools-plugin/src/com/google/cloud/tools/intellij/appengine/sdk/CloudSdkService.java

```diff
@@ -39,6 +40,8 @@ public static CloudSdkService getInstance() {
 
   public abstract void setSdkHomePath(String path);
 
+  public abstract Set<CloudSdkValidationResult> validateCloudSdk(@NotNull Path pathToCloudSdk);
```

Comment:
Pretty cool, even though I would really like to see something like this actually happening at the tools library. (and meant to do so whenever I had some free time)